de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/appdict/validation/model/validator/AD_Column.java

```diff
@@ -0,0 +1,17 @@
+package org.adempiere.appdict.validation.model.validator;
+
+import org.adempiere.ad.modelvalidator.annotations.Interceptor;
+import org.adempiere.ad.modelvalidator.annotations.ModelChange;
+import org.compiere.model.I_AD_Column;
+import org.compiere.model.ModelValidator;
+
+@Interceptor(I_AD_Column.class)
+public class AD_Column
+{
+	@ModelChange(timings = { ModelValidator.TYPE_BEFORE_NEW }, ifColumnsChanged = { I_AD_Column.COLUMNNAME_Updated, I_AD_Column.COLUMNNAME_UpdatedBy })
```

Comment:
A user might create a new record and edit it to their needs, and then press save and the flag is silently unset, if it was set by the user..
i think that's not so good.

please rather modify the process that copies new AD_Columns from other tables (because imho that's how 95% of neww Updated* columns are made).
And consider using a tab callout for new ad_columns that are created by a user.

Reply:
or just change the default logic to "N"

Reply:
update: as talked with @metas-ts we need to set IsAllowLogging=N only for Created/Updated and CreatedBy/UpdatedBy. For that i think we need a callout which is triggered when ColumnName (or AD_Element_ID).